### PR TITLE
Travis CI fixes

### PR DESCRIPTION
**Pull request by isuruf**

### CODE REVIEW

**Comment by isuruf** [comment]
@certik, @CodeMaxx, can you take a look?

**Comment by CodeMaxx** [comment]
@isuruf I'm trying to fix the same think in #1092 . Take a look at https://github.com/travis-ci/travis-ci/issues/6307

**Comment by isuruf** [comment]
Yes, that travis error is there. I can't debug as travis is backlogged on OSX for more than a day now.

**Comment by CodeMaxx** [comment]
This might help https://github.com/schwa/SwiftIO/commit/f9914fabda2ee1509bef5110f2028efc424d828d

File: bin/install_travis.sh [hunk 1]
```diff
@@ -97,8 +97,12 @@ if [[ "${WITH_MPC}" == "yes" ]]; then
...
-    cd piranha && mkdir build && cd build;
+    wget https://github.com/bluescarni/piranha/archive/v0.5.tar.gz;
+    tar -xzf v0.5.tar.gz;
+    cd piranha-0.5 && mkdir build && cd build;
     cmake -DCMAKE_BUILD_TYPE=Release -DCMAKE_INSTALL_PREFIX=$our_install_dir -DBUILD_TESTS=no ../ && make -j8 install && cd ../..;
 fi
 cd $SOURCE_DIR;
+
+# Since this script is getting sourced, remove error on exit
+set +e
```
Comment (certik) [inline bin/install_travis.sh]: Indeed you have to do it if it is sourced. But why is it sourced?

File: .travis.yml [hunk 2]
```diff
@@ -67,13 +67,16 @@ matrix:
...
           - llvm-toolchain-precise-3.7
+          - george-edison55-precise-backports
           packages:
           - libgmp-dev
           - libmpfr-dev
           - libmpc-dev
           - binutils-dev
           - g++-5
           - clang-format-3.7
+          - cmake
```
Comment (CodeMaxx) [inline .travis.yml]: Is this change necessary even though we are using an older piranha version now?

Comment (isuruf) [inline .travis.yml]: yes, piranha-v0.5 needs cmake >= 3.0

File: bin/install_travis.sh (hunk 1 above)
Comment (isuruf) [inline bin/install_travis.sh]: Because this script sets up the environment for the other scripts

**Comment by isuruf** [comment]
@CodeMaxx, @certik, can you review? This PR is blocking others.

**Comment by CodeMaxx** [comment]
@isuruf Made a comment. The rest looks good to me.

**Comment by certik** [comment]
This is +1 to merge from me. Thanks @isuruf !

**Comment by isuruf** [comment]
@certik, @CodeMaxx thanks for the review
